src/System.Text.Encoding/tests/UTF32Encoding/UTF32EncodingDecode.cs

```diff
@@ -101,28 +101,28 @@ public void Decode_InvalidBytes()
             Decode_InvalidBytes(new byte[] { 0x00, 0x00, 0x00, 0xFF }, 0, 4, "\uFFFD");
             Decode_InvalidBytes(new byte[] { 0xFF, 0xFF, 0xFF, 0xFF }, 0, 4, "\uFFFD");
         }
-
-        public static byte[] GetBigEndianBytes(byte[] littleEndianBytes)
+        
+        public static byte[] GetBigEndianBytes(byte[] littleEndianBytes, int index, int count)
```

Comment:
The previous implementation is broken for non zero indexes or counts. I fixed UnicodeEncoding's implementation, so fixed Utf32Encoding's implementation too just so no weird errors occur if we add tests for invalid bytes in non zero indexes